README.md

```diff
@@ -120,6 +121,7 @@
 - [Dani Akash](https://github.com/daniakash/daniakash)
 - [Rahul Jha](https://github.com/rahul-jha98/rahul-jha98)
 - [Taehyun Hwang](https://github.com/HwangTaehyun/HwangTaehyun)
+- [HsiangNianian](https://github.com/HsiangNianian/HsiangNianian)
```

Comment:
Why duplicate?

Reply:
Excuse me, what do you mean? Can you clarify that? 🤗

Reply:
sure, line 124 & 86. Do your profile really should be in two places?